docs/website/docs/user-guides/data-scientist.md

```diff
@@ -0,0 +1,156 @@
+# Data Scientist
+
+Data Load Tool (`dlt`) can be highly useful for Data Scientists in several ways. Here are four
+potential use cases:
+
+## Use case #1: Efficient Data Ingestion and Optimized Workflow
+
+Data Scientists often deal with large volumes of data from various sources. `dlt` can help
+streamline the process of data ingestion by providing a robust and scalable tool for loading data
+into their analytics environment. It can handle diverse data formats, such as CSV, JSON, or database
+dumps, and efficiently load them into a data lake or a data warehouse.
+
+![dlt-main](images/dlt-main.png)
+
+By using `dlt`, Data Scientists can save time and effort on data extraction and transformation
+tasks, allowing them to focus more on data analysis and models training. The tool is designed as a
+library that can be added to their code, making it easy to integrate into existing workflows.
+
+`dlt` can facilitate a seamless transition from data exploration to production deployment. Data
+Scientists can leverage `dlt` capabilities to load data in the format that matches the production
+environment while exploring and analyzing the data. This streamlines the process of moving from the
+exploration phase to the actual implementation of models, saving time and effort. By using `dlt`
+throughout the workflow, Data Scientists can ensure that the data is properly prepared and aligned
+with the production environment, leading to smoother integration and deployment of their models.
+
+- Use existed Verified Sources and pipeline examples or create your own quickly. See how to
+  [create pipeline](../walkthroughs/create-a-pipeline).
+
+- [Deploy this pipeline](../walkthroughs/deploy-a-pipeline), so that the data
+  is automatically loaded on a schedule.
+
+- Transform the [loaded data](../dlt-ecosystem/transformations/transforming-the-data) with
+  dbt or in Pandas DataFrames.
+
+- Learn how to [run](../running-in-production/running),
+  [monitor](../running-in-production/monitoring), and
+  [alert](../running-in-production/alerting) when you put your pipeline in
+  production.
+
+- Use `dlt` when doing exploration in a Jupyter Notebook and move more easily to production. Explore
+  our
+  [Colab Demo for Chess.com API](https://colab.research.google.com/drive/1NfSB1DpwbbHX9_t5vlalBTf13utwpMGx?usp=sharing)
+  to realize how easy it is to create and use `dlt` in your projects:
+
+  ![colab-demo](images/colab-demo.png)
+
+### `dlt` is optimized for local use on laptops
+
+- It offers a seamless
+  [integration with Streamlit](../dlt-ecosystem/visualizations/understanding-the-tables#show-tables-and-data-in-the-destination).
+  This integration enables a smooth and interactive data analysis experience, where Data Scientists
+  can leverage the power of `dlt` alongside Streamlit's intuitive interface and visualization
+  capabilities.
+- In addition to Streamlit, `dlt` natively supports
```

Comment:
there's share-a-dataset where we explain how to go from local to bigquery. maybe we should link it?